lib/edit/MonsterRaceDefinitions.txt

```diff
@@ -27140,3 +27140,53 @@ D:$It is seldom found but greatly formidable.
 D:$  You'll be defeated without complete preparation.
 D:それはめったに出会えないが、恐ろしく強力な力を持っている。
 D:生半可な態勢で挑めばあっという間に返り討ちにされるだろう。
+
+# 特殊仕様で以下の特性あり
+# ・一度に受けるダメージは現在HPの10%まで。
+# ・ただしこの軽減効果は最大HPの1.4%を下回らない。
+# ・救援召喚はウサウサストライカー（後述）を
+#   2体召喚する。
+# このダメージキャップは最低29回攻撃を受けないと死なない。
+# 元ネタにはある延命がないため、その分ダメージキャップは盛っている。
+# （元々は33回だったが、90F前提のものなため調整で弱体化した）
+# （ちなみに元ネタでは31回）
+N:1355:ラフィーII
+E:Laffey II
+G:g:w
+I:125:50d40:40:180:30
+W:63:6:35000:0:0
+B:SHOOT:HURT:12d12
+B:PUNCH:SUPERHURT:12d12
+B:HIT:HURT:8d8
+B:KICK:HURT:8d8
+X:FEMALE
+F:ONLY_ONE | NONLIVING | GOOD | FORCE_MAXHP | SPEAK_ALL |
+F:WEIRD_MIND | LAFFEY_II |
+F:IM_FIRE | IM_COLD | IM_POIS | RES_DARK | RES_LITE | RES_DISE | RES_CHAO | RES_WATE |
+F:NO_CONF | NO_FEAR | NO_INSTANTLY_DEATH |
+F:OPEN_DOOR | BASH_DOOR | MOVE_BODY | TAKE_ITEM | CAN_SWIM | SMART |
+F:DROP_CORPSE | DROP_GREAT | DROP_2D2 |
+S:1_IN_2
+S:ROCKET | PSY_SPEAR | BO_LITE | SHOOT | S_KIN |
+R:1356:2d1
+D:$Laffey became extremely stronger...
+D:$ Yep, became stronger... Zzzzz...
+D:ラフィーとっても強くなった…
+D:うん、強くなった……Zzzz…
+
+N:1356:ウサウサストライカー
+E:Bunbun Striker
+G:*:w
+I:117:25d22:50:90:0
+W:63:15:5000:0:0
+F:NONLIVING | FORCE_MAXHP | EMPTY_MIND | SMART |
+F:CAN_FLY | RAND_25 | NEVER_BLOW |
+F:HURT_FIRE | IM_COLD | IM_POIS | RES_DARK | RES_LITE | RES_DISE | RES_CHAO | RES_WATE |
+F:NO_CONF | NO_FEAR | NO_SLEEP | NO_STUN |
+S:1_IN_4
+S:PSY_SPEAR | BO_LITE |
+D:$It is a part of the Laffey II's rigging, and bunny-looking, propeller-less, and futuristic drone.
```

Comment:
I'd use

```
D:$It is a part of Laffey II's rigging: a bunny-looking, propeller-less, futuristic drone.
D:$ It can project light bolts that burn a hole through a target.
```